cfme/tests/control/test_bugs.py

```diff
@@ -58,7 +59,7 @@ def test_scope_windows_registry_stuck(request, setup_a_provider):
     vm.assign_policy_profiles(profile.description)
     # It should be screwed here, but do additional check
     pytest.sel.force_navigate("dashboard")
```

Comment:
Check out cfme.base.ui - there's a Dashboard destination that you may be able to utilize here to remove another force_navigate.

Possibly outside of the intended scope of the PR, at your discretion.